packages/bundle/src/adaptiveCards/Styles/adaptiveCardHostConfig.js

```diff
@@ -1,4 +1,4 @@
-import defaultStyleSetOptions from './defaultStyleSetOptions';
+import defaultStyleOptions from 'botframework-webchat-component/src/Styles/defaultStyleOptions';
```

Comment:
Don't reference it under `/src/`.

```suggestion
import { defaultStyleOptions } from 'botframework-webchat-component';
```